### DISCUSSION TITLE
PS1 doesn't play

### DISCUSSION THREAD

**Discussion by David31o5** [OnionUI/Onion#491] [General]
Hello, sorry if it's an issue that was discussed befor, I followed all the installion process. everything works ' only ps1 not' I moved the system folder to the BIOS folder and of course all the roms, what can be the reason?
Thanks, David

**Comment by David31o5** [comment]
After refreshing the files it started to work.

**Reply by pocket-books** [reply]
Would you please explain the refresh?
I also have moved the contents of the retroarch 'system' folder into the surface level 'BIOS' folder.
When I go into a core and do a search of all 9000 bios files it still does not recognize any of my scphXXXX.bios files.
- I've checked the MD5 checksums as well and they are all the ones included in the libretro guide.
